lib/y2logsstep.pm

```diff
@@ -341,6 +339,7 @@ sub post_fail_hook {
 
     # Collect yast2 installer  strace and gbd debug output if is still running
     $self->save_strace_gdb_output;
+    $self->problem_detection;
```

Comment:
This is a real killer. The "problem_detection" is good but together with all the additional debugging information recorded, e.g. even the "strace and gdb" output just one line above whenever a yast module fails it will take very very long to finish. This can cause a lot of incompletes because of jobs running into the MAX_JOB_TIME while executing the post_fail_hook. Please do not do that. It would be great to reuse some of the stuff from the problem_detection method but this is not the right way.